client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
@ksteph - If we're assuming that students shouldn't modify our files, yes. What I was trying to do is basically remove the "Case #" and "Suite #" part of the question ID without having any issues. If that's not a concern, then the current question ID is fine.